go11x/static fails with Error: Server Error

```
$ git clone https://github.com/GoogleCloudPlatform/golang-samples.git
$ cd golang-samples/appengine/go11x/static
```
ok
```
$ go version
go version go1.13 linux/amd64
$ go run .
2019/09/20 17:51:01 Defaulting to port 8080
2019/09/20 17:51:01 Listening on port 8080
```
ok, local server works fine
```
$ gcloud app deploy
```
ok, provided some account and project are set, and some Cloud Build API is enabled.

```
$ gcloud app browse
```
**Error: Server Error
The server encountered an error and could not complete your request.
Please try again in 30 seconds.**

In the Stackdriver Loggind web view:

```
panic: open templates/index.html: no such file or directory
```

--- a/appengine/go11x/static/go.mod
+++ b/appengine/go11x/static/go.mod
@@ -1,3 +1,3 @@
 module github.com/GoogleCloudPlatform/golang-samples/appengine/go11x/static
 
-go 1.13
+go 1.11

--- a/appengine/go11x/static/go.mod
+++ b/appengine/go11x/static/go.mod
@@ -1,0 +1,3 @@
+module github.com/GoogleCloudPlatform/golang-samples/appengine/go11x/static
+
+go 1.13

--- a/internal/e2e/standard_test.go
+++ b/internal/e2e/standard_test.go
@@ -40,6 +40,19 @@ func TestHelloWorld(t *testing.T) {
 	bodyShouldContain(t, helloworld, "/", "Hello, World!")
 }
 
+func TestGo11xStatic(t *testing.T) {
+	tc := testutil.EndToEndTest(t)
+
+	helloworld := &aeintegrate.App{
+		Name:      "static",
+		Dir:       tc.Path("appengine", "go11x", "static"),
+		ProjectID: tc.ProjectID,
+	}
+	defer helloworld.Cleanup()
+
+	bodyShouldContain(t, helloworld, "/", "The Gopher Network")
+}
+
 func bodyShouldContain(t *testing.T, p *aeintegrate.App, path, shouldContain string) {
 	if p.Deployed() {
 		t.Fatalf("[%s] expected non-deployed app", p.Name)

